src/ddmd/mars.d

```diff
@@ -43,7 +43,7 @@ import ddmd.json;
 import ddmd.lib;
 import ddmd.link;
 import ddmd.mtype;
-import ddmd.objc;
+import objc = ddmd.objc;
```

Comment:
I'll remove this.

It's a bit difficult to know which D features can be used. Are there some guidelines for DMD somewhere?